NEWS

```diff
@@ -1,5 +1,21 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+New in version 3
+
+* New man page format: single-commands-section
+* Add CLI and conf options for setting the output format
+* Add CLI and conf options to explicitly specify %prog value
+* Skip showing commands with help == SUPPRESS
+* Avoid rendering docs for command aliases
+* Print program name in upper case in header and footer
+* Dropped --author-email CLI option
```

Comment:
I will probably re-add this option, just to stay compatible with previous releases JFYI.